brave-kafka/src/main/java/smartthings/brave/kafka/consumers/BaseTracingConsumerInterceptor.java

```diff
@@ -89,6 +85,19 @@
     return new ConsumerRecords<>(tracedRecords.getRecords());
   }
 
+  protected void reportServerReceivedSpan(TraceContext traceContext, ConsumerRecord<K, byte[]> record) {
+    // Only report "sr" if the span was sampled on client side;
+    // reporting only "sr" will result in NaN duration in the zipkin UI
+    if (traceContext != null && traceContext.sampled() == Boolean.TRUE) {
+      tracer.joinSpan(traceContext)
+        .name(nameProvider.spanName(record))
+        .remoteEndpoint(remoteEndpoint)
+        .annotate(Constants.SERVER_RECV)
```

Comment:
Setting this annotation explicitly shouldn't be necessary if you set the kind `Span.Kind.SERVER` and call call `start()`.  Also I wonder if the NaN duration is related to not calling `start()`?  Or maybe thats just a bug in the UI.